apps/andi/lib/andi/input_schemas/dataset_input.ex

```diff
@@ -51,7 +52,8 @@ defmodule Andi.InputSchemas.DatasetInput do
     :private,
     :publishFrequency,
     :sourceFormat,
-    :sourceType
+    :sourceType,
+    :sourceUrl
```

Comment:
Can you add to the `data_test` that we have for required fields in `dataset_input_test`?

Reply:
I believe I did. Can you point out the specific test?

Reply:
Oh yeah, you did.  I apparently don't know how to read diffs.  Sorry!